app/konnect/getting-started/index.md

```diff
@@ -2,60 +2,50 @@
 title: Get started with Konnect
 ---
 
-## Quickstart guide
+{% mermaid %}
+flowchart TD
+    A{Start my {{site.konnect_short_name}} journey} -->B(Set up an API for me)
+    A{Start my {{site.konnect_short_name}} journey} -->C(I want to do it myself)
+    B --> D(Use the onboarding wizard for the first time)
+    C --> E(Add an API to {{site.konnect_short_name}})
+    C --> F(Import {{site.base_gateway}} entities to {{site.konnect_short_name}})
 
-New to {{site.konnect_saas}}? Get started with the basics through the web app:
+    %% this section defines node interactions
+    click D "https://cloud.konghq.com/quick-start"
+    click E "/konnect/getting-started/add-api/"
+    click F "/konnect/getting-started/import/"
+{% endmermaid %}
 
-1. **Create a {{site.konnect_short_name}} account**:
+## About {{site.konnect_short_name}}
```

Comment:
❗ For reviewers: Connor requested a small section here that covers some concepts and architecture. 
* Do you think this works? Or do you think adding content here might detract from the getting started flow? 
* Could I get a tech accuracy check on diagrams and words to make sure everything I said is accurate?
* I'm also worried that this is kind of duplicating content from the intro/architecture pages, but I made the shared content into includes. What do you think?

Reply:
honestly, I like it. The pathway diagram with links is still at the top of the page, so it's the first eye-catching thing anyway, and then they can choose to learn more about Konnect if they want to.